Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/DXResourceBinding.cpp

```diff
@@ -0,0 +1,59 @@
+#define NOMINMAX
+#define WIN32_LEAN_AND_MEAN
+
+#include "pch.h"
+#include "DXResourceBinding.h"
+#include "DXResourceBinding.g.cpp"
+#include "stdafx.h"
+#include "ORTHelpers.h"
+
+using namespace winrt::Microsoft::AI::MachineLearning;
+
+#undef min
```

Comment:
🤔 shouldn't be necessary given you have `NOMINMAX` above.

Reply:
Done.